app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -453,6 +455,9 @@ class MessagingManager @Inject constructor(
                             }
                         }
                     }
+                    COMMAND_APP_LOCK -> {
+                        handleDeviceCommands(jsonData)
```

Comment:
Are all of the new parameters mean to be optional? Do we need to validate the data coming in for each one?